core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -767,6 +767,11 @@ public void truncateTable(String namespace, String table) throws ExecutionExcept
   public void createIndex(
       String namespace, String table, String columnName, Map<String, String> options)
       throws ExecutionException {
+    if (getTableMetadata(namespace, table).getColumnDataType(columnName) == DataType.BOOLEAN) {
+      throw new IllegalArgumentException(
+          "Currently, BOOLEAN type is not supported for a secondary index in DynamoDB: "
+              + columnName);
+    }
 
```

Comment:
I added this check because the message of the exception thrown by the Dynamo SDK is not clear enough boolean index are not supported.